src/Client.js

```diff
@@ -866,21 +868,62 @@ class Client extends EventEmitter {
             );
         }
 
+
+        if (internalOptions.attachment?.data?.length > (1024 * 1024 * 79)) {
```

Comment:
attachment.size does not get fullfilled if the developer creates his own messagemedia out of a base64 string

Reply:
> attachment.size does not get fullfilled if the developer creates his own messagemedia out of a base64 string (and it used to not get fullfilled when using MessageMedia.fromFilePath as well, but i updated that function)

So why don't we added a condition that if filesize not exist, set filesize to the base64 length as well?
If we have size parameter, why not use it?

Reply:
I still get a bit skeptical and try to avoid "changing more than necessary"

I've implemented your idea now, tested on a few medias that i generate from base64, as well as doing the same tests as previously (big files, small files, stickers, videos, audios, documents)

Thanks!